torch/fx/experimental/symbolic_shapes.py

```diff
@@ -270,7 +270,7 @@ def _nyi():
     'neg',
 }
 
-float_magic_methods = {"add", "sub", "mul", "truediv", "ceil", "floor", "eq", "gt", "lt", "le", "ge", "pow"}
+magic_methods_not_on_float = {"sym_int"}
```

Comment:
FWIW, SymFloat should have `__sym_int__`, it just needs to be implemented (and figure out why the obvious Sympy implementation doesn't work)